packages/ra-ui-materialui/src/form/SimpleForm.spec.js

```diff
@@ -8,7 +8,7 @@ import TextInput from '../input/TextInput';
 describe('<SimpleForm />', () => {
     it('should embed a form with given component children', () => {
         const wrapper = shallow(
-            <SimpleForm>
+            <SimpleForm handleSubmit={() => {}}>
```

Comment:
If this is required, then your PR is a BC break

Reply:
The `SimpleForm` in the tests is the not decorated by redux-form one. `handleSubmit` is injected by redux-form

Reply:
ok then